cmd/bk/main.go

```diff
@@ -18,13 +18,18 @@ func main() {
 
 func mainRun() int {
 	ctx := context.Background()
-	viper := viper.New()
-	viper.SetConfigFile(config.ConfigFile())
-	viper.AutomaticEnv()
+	v := viper.New()
+	v.SetConfigFile(config.ConfigFile())
+	v.AutomaticEnv()
 	// attempt to read in config file but it might not exist
-	_ = viper.ReadInConfig()
+	if err := v.ReadInConfig(); err != nil {
+		if _, ok := err.(viper.ConfigFileNotFoundError); ok {
+			fmt.Print(fmt.Errorf("Unable to find config file at %s: %s", config.ConfigFile(), err))
+			return 1
+		}
+	}
```

Comment:
I thought about this handling but that means the config file has to exist first which makes the configure command kinda redundant. We should be able to operate the entire bk cli without it

Reply:
It also handles the fact that between `configure` and `<command>` the user has deleted their config file.

Reply:
I've checked this out locally because it wasnt making sense. If the file doesn't exist it should error out here and make the CLI unusable. But it wasn't doing that.

Turns out that's because the err type is actually `*fs.PathError` so it never gets into the if condition to exit early.

But we don't want that anyway. We need to be able to invoke the command in order to create the config file. So the config file not existing is valid